src/installer/tests/Assets/TestProjects/HostApiInvokerApp/HostFXR.cs

```diff
@@ -24,27 +24,27 @@ internal enum hostfxr_resolve_sdk2_result_key_t : int
                 global_json_path = 1,
             }
 
-            [StructLayout(LayoutKind.Sequential, CharSet = Utils.OSCharSet)]
+            [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
             internal struct hostfxr_dotnet_environment_sdk_info
             {
-                internal int size;
+                internal UIntPtr size;
```

Comment:
This should be `nuint` - which is `UIntPtr` but conveys the desired idea of `size_t`.